lib/watcher.js

```diff
@@ -10,6 +10,11 @@ function Watcher(builder, options) {
   this.options = options || {}
   this.watchedDirs = {}
 
+  // notify the first build once the object is returned
+  process.nextTick(function(){
+    this.emit('start')
+  }.bind(this))
+
```

Comment:
No.
I don't remember the exact reason (sorry, but I sent this couple of weeks ago XD) but on start up that code is skipped. Probably because no dirs have changed and nothing is being watched. That's why I had to handle that case separately.